src/twilio/functions/sms/intake-delivery-txts.js

```diff
@@ -0,0 +1,99 @@
+exports.handler = function codeSms(context, event, callback) { // eslint-disable-line
+  const Airtable = require("airtable"); // eslint-disable-line
+  Airtable.configure({
+    endpointUrl: "https://api.airtable.com",
+    apiKey: context.AIRTABLE_API_KEY // eslint-disable-line
+  });
+
+  const base = Airtable.base("apppK7mrvMPcwtv6d"); // intake + vols base
+  let code = event.body.requestCode;
+  const deliveryPhone = event.body.deliveryPhone; // eslint-disable-line
+  let deliveryName = event.body.deliveryName; // eslint-disable-line
+  let intakePhone = event.body.intakePhone; // eslint-disable-line
+  let intakeName = event.body.intakeName; // eslint-disable-line
+  let body = JSON.stringify(event.body); // eslint-disable-line
+  let missinginfo = false;
```

Comment:
just a nit but usually in JS variables should be camelCased

Reply:
🤖✅